travis-ci/vmtest/configs/BLACKLIST

```diff
@@ -0,0 +1,3 @@
+LATEST	mmap,dctcp,cubic,bpf_tcp_ca,bpf_verif_scale,cgroup_attach,pinning,send_signal_tracepoint_thread,test_syncookie,select_reuseport,send_signal,sockopt_inherit,strobemeta_nounroll2,stacktrace_build_id,tp_attach_query,tcp_rtt,task_fd_query_tp,stacktrace_map,fentry,test_overhead,kfree_skb,fexit_stress,fexit_test
```

Comment:
this seems to be rather complicated format. I think having one file per each version would be much simpler and easier to work with. E.g., BLACKLIST-latest, BLACKLIST-5.5.0, etc. WDYT?